ui/src/vayla-design-lib/dropdown/dropdown.tsx

```diff
@@ -287,7 +287,7 @@ export const Dropdown = function <TItemValue>({
                 }
             }
         }
-    }, [open]);
+    }, [open, optionFocusIndex]);
```

Comment:
Aiemmin (as in silloin kun tämä vielä toimi) tämä ajettiin joka renderillä. Sittemmin tämä scrollaus laitettiin ajautumaan ainoastaan haulla, mutta tämähän pitää ajaa myös kun vaihtoehtojen välillä hypitään näppiksellä